Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buoyantio/linkerd:1.4.3
+FROM buoyantio/linkerd:1.5.0
 
 RUN mkdir -p $L5D_HOME/plugins
 COPY plugins/*.jar $L5D_HOME/plugins
```

Comment:
I have a newer version of the docker CLI installed, and this COPY command fails for me, with:

```
Step 3/3 : COPY plugins/*.jar $L5D_HOME/plugins
When using COPY with more than one source file, the destination must be a directory and end with a /
```

Would you mind updating it while you're in here? This works:


```diff
diff --git a/Dockerfile b/Dockerfile
index c220601..7315180 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
 FROM buoyantio/linkerd:1.5.0
 
 RUN mkdir -p $L5D_HOME/plugins
-COPY plugins/*.jar $L5D_HOME/plugins
+COPY plugins/*.jar $L5D_HOME/plugins/
```

---

```
$ docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
 Kubernetes:
  Version:          v1.10.3
  StackAPI:         v1beta2
```

Reply:
Thanks for the diff! I wasn't aware of that change. Should have run a docker build to test.